.pre-commit-config.yaml

```diff
@@ -0,0 +1,21 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.5.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-added-large-files
+-   repo: https://github.com/pycqa/flake8
+    rev: 7.0.0
+    hooks:
+    -   id: flake8
+        additional_dependencies: [flake8-docstrings]
```

Comment:
It looks like this isn't installed in the `pyproject.toml` file. We can add it there, or remove the `flake8-docstrings` here and update both later. It currently raises errors about docstrings, which doesn't happen when just `flake8 .` is run.

Reply:
@jtyoung84 , sorry for the delay. This is what you had in mind, correct? https://github.com/AllenNeuralDynamics/aind-metadata-service/pull/371/commits/23c6cbcaa7bc4018f68b0af777242f8c6e1d5113

Reply:
That was one suggestion. Looks good.